clang/include/clang/StaticAnalyzer/Core/PathSensitive/EntryPointStats.h

```diff
@@ -0,0 +1,162 @@
+// EntryPointStats.h - Tracking statistics per entry point --*- C++ -*-//
```

Comment:
The line length seems odd. It's much smaller than 81 characters.
Same observation goes at the end of this comment block.

Reply:
Fixed
a93cac41b87d [NFC] Fix header comment widths